client/package.json

```diff
@@ -236,7 +236,7 @@
 		"@testing-library/react": "^15.0.7",
 		"@testing-library/user-event": "^14.5.2",
 		"@types/react": "^18.2.14",
-		"@types/redux-mock-store": "1.0.6",
+		"@types/redux-mock-store": "1.5.0",
```

Comment:
Not for this PR, but ideally, we should stop using `redux-mock-store`, see https://redux.js.org/usage/writing-tests#setting-up-a-reusable-test-render-function 

Care to open an issue about it?

Reply:
I think that would be a good idea because it will improve the tests. I have created https://github.com/Automattic/wp-calypso/issues/98890, the problem is that it involves a few files, around 77.

Reply:
Thanks! Should be straightforward. Also, easy to confirm it works with passing tests.

Reply:
Great. Thanks for reviewing it. I will merge this and follow up on the issue created by @epeicher.